examples/plot_metricframe_beyond_binary_classification.py

```diff
@@ -0,0 +1,245 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+=========================================
+MetricFrame: Beyond Binary Classification
+=========================================
+"""
+
+# %%
+# This notebook contains examples of using :class:`~fairlearn.metrics.MetricFrame`
+# for tasks which go beyond simple binary classification.
+
+import sklearn.metrics as skm
+import functools
+from fairlearn.metrics import MetricFrame
+
+# %%
+# Multiclass & Nonscalar Results
+# ==============================
+#
+# Suppose we have a multiclass problem, with labels :math:`\in {0, 1, 2}`,
+# and that we wish to generate confusion matrices for each subgroup
+# identified by the sensitive feature :math:`\in { a, b, c, d}`.
+# This is supported readily by
+# :code:`~fairlearn.metrics.MetricFrame`, which does not require
+# the result of a metric to be a scalar.
+#
+# First, let us generate some random input data:
+
+import numpy as np
+
+rng = np.random.default_rng()
```

Comment:
I would get an rng with a seed to have the notebook reproducible

Reply:
Done